packages/nodejs-ext/scripts/extension.js

```diff
@@ -71,8 +71,7 @@ function download(mirrors, filename, outputPath) {
     downloadFromMirror(mirrors.shift(), filename, outputPath)
       .then(url => {
         const downloadData = {
```

Comment:
I think we don't need `downloadData` to be an object anymore. We can just return the URL

Reply:
Good point! I removed the object.